packages/quantic/force-app/main/default/lwc/quanticColoredResultBadge/quanticColoredResultBadge.js

```diff
@@ -20,6 +19,23 @@ export default class QuanticColoredResultBadge extends LightningElement {
    * @type {string}
    */
   @api label;
+  /**
+   * The [result item](https://docs.coveo.com/en/headless/latest/reference/search/controllers/result-list/#result).
+   * @api
+   * @type {Result}
+   */
+  @api result;
+  /**
+   * The [result field](https://docs.coveo.com/en/headless/latest/reference/search/controllers/result-list/#result).
+   * @api
+   * @type {string}
+   */
+  @api field;
+  /**
+   * The field type whose values you want to display.
+   * @type { 'string' | 'date' | 'number' | 'multi'}
+   */
+  @api type;
```

Comment:
"type" ... because it has "type", do we need to provide function the format the value?

Reply:
This component uses the other components (quantic-result-[text,date,number,multi-value-text]) under the hood so they handle the formatting already.

Reply:
I am not sure to properly understand this sentence. The user needs to provide a value.

For instance, `type="string"`

This is based on this component: [quanticResultField](https://github.com/coveo/ui-kit/blob/master/packages/quantic/force-app/main/default/lwc/quanticResultField/quanticResultField.js)